app/Http/Transformers/SignupTransformer.php

```diff
@@ -26,6 +26,7 @@ class SignupTransformer extends TransformerAbstract
      */
     public function transform(Signup $signup)
     {
+        dd($signup->events);
```

Comment:
ok thanks, Dave!! I will give that a shot and yes would love to chat more IRL about the relationships!